ecs-cli/modules/commands/compose/service/compose_service_command.go

```diff
@@ -180,6 +181,10 @@ func loadBalancerFlags() []cli.Flag {
 			Name:  flags.RoleFlag,
 			Usage: roleUsageString,
 		},
+		cli.StringFlag{
+			Name:  flags.HealthCheckGracePeriodFlag,
```

Comment:
The flag name is descriptive but very long. I'm wondering if it would be a good idea to add an alias for it, like `--hcgp`. What do you think?

Reply:
It is long, though it's shorter than the existing "--deployment-min-healthy-percent" flag. Given that and the fact we don't currently alias any flags, my preference is to stay consistent. 

Maybe in the future we can add truncated flag options for the longest offenders so customers can chose which to use?

Reply:
I am okay with the full name, as it maps directly to what the field is called on the backend. We can discuss names for shorter aliases later.